src/components/Tables/Table.tsx

```diff
@@ -256,11 +238,10 @@ const Table: React.FC<Props> = ({
         });
     }
 
-    // TODO VFB-23 Implement conditional styling: center icon when only option selected, grid otherwise
-    if (reorderable || onEdit || onDelete) {
+    if (reorderable || onEdit) {
         columns.unshift({
-            name: "",
-            cell: (row: Row) => {
+            name: <p>Sort</p>,
```

Comment:
Why is it named sort? I think the column does more than sorting though there is also edit and delete

Reply:
It feels like the most intuitive name, feel free to suggest another